lib/workers/bin/run_single_worker.rb

```diff
@@ -42,7 +46,6 @@
   exit
 end
 opt_parser.abort(opt_parser.help) unless worker_class
-
```

Comment:
WHY!!!1!?!  :rage4:

Reply:
Oh that'll be me putting in logging and when I removed it i've changed the formatting. That or Atom did it. Yeah I'll blame Atom.

Reply:
lolz